### ISSUE TITLE
Octo didn't fully load for the first time in a new browser

### ISSUE THREAD

**Issue by wbconnor** [voraciousdev/octo#14]
Opened Octo in Chrome `84.0.4147.135` on Mac `10.14.6` a blank page loaded, ready to start taking a note. The side bar and actions button did not load. The welcome note did not load. I started typing a note and it would not save. I tried to hit a login route by going to `octo.app/login` got a 404 (understandable) and then reloaded the root route and the page loaded as expected.

**Comment by wbconnor** [comment]
Possibly separate issue: I created a new note, and then signed in. Octo didn't load the rest of my notes or tags until I made a change on that new note. Before doing that I also tried reloading the page.

![Screen Shot 2020-08-27 at 10 43 44 AM](https://user-images.githubusercontent.com/8883983/91457644-92700a80-e852-11ea-9fed-d1a6c6ee3da3.png)


![Screen Shot 2020-08-27 at 10 44 23 AM](https://user-images.githubusercontent.com/8883983/91457654-969c2800-e852-11ea-8105-dcef98196867.png)
